NitroxServer/GameLogic/Entities/Spawning/BatchEntitySpawner.cs

```diff
@@ -282,9 +314,33 @@ private void AssignPlaceholderEntitiesIfRequired(Entity entity, string classId,
                                             deterministicBatchGenerator.NextId(),
                                             entity);
 
+                    if (prefab.EntitySlot.IsPresent())
+                    {
+                        Entity possibleEntity = SpawnEntitySlotEntities(prefab.EntitySlot.Get(), transform, deterministicBatchGenerator, entity);
+                        if (possibleEntity != null)
+                        {
+                            entity.ChildEntities.Add(possibleEntity);
```

Comment:
Should we add all of the entities that are spawned in `SpawnEntitySlotEntities`?  From your logic, it looks like you only add the first.  Is there significance to that?

Reply:
there should only be ONE entity being returned which is always the first one, the rest of the returned entities are children